dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -752,10 +752,10 @@ export const Card = ({
 	return (
 		<CardWrapper
 			format={format}
-			showTopBarDesktop={!isOnwardContent && showTopBarDesktop}
```

Comment:
The Carousel calling the card component now specifies that there shouldn't be a top bar at both screen sizes when the card is Onwards Content, so this extra check isn't needed.